src/main/java/com/entropyteam/entropay/employees/clients/models/Client.java

```diff
@@ -0,0 +1,49 @@
+package com.entropyteam.entropay.employees.clients.models;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Table(name = "clients", schema = "employees")
```

Comment:
are we having different schemas depending on the table? maybe better to set a default schema on the connection?

Reply:
I agreed from the beginning. @agustinb4 Maybe we need to re-discuss this?